spec/srb_json_load_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+require './lib/models/driver'
+
+describe 'srb_json_load' do
+  Dir['./**/*.json'].each do |json_file|
+    next if json_file.include? 'jtd.json'
+
+    it json_file.to_s do
+
+      data = nil
+      if json_file.include? 'lib/engines/hcktest/drivers'
+        data = AutoHCK::Models::Driver.from_json_file(json_file)
+      else
+        pending("NO TYPE FOR #{json_file}")
+        raise
+      end
```

Comment:
```suggestion
      unless json_file.include? 'lib/engines/hcktest/drivers'
        pending("NO TYPE FOR #{json_file}")
        raise
      end

      data = AutoHCK::Models::Driver.from_json_file(json_file)
```

Reply:
I am not sure about this. In the next commit, we will add platforms model and add `if json_file.include? 'lib/engines/hcktest/platforms'` or do you have another idea of how to select the proper type for JSON?

Reply:
What about having an array typed `T::Array[[String, AutoHCK::Models::JsonHelper]]`, where the `String` is a glob pattern and `AutoHCK::Models::JsonHelper` is the corresponding model?